### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AdornChoga**
In this pull request, we:

- Created `decode_char` method for decoding each character
- Created `decode_word` method for decoding each word
- Created `decode` method for decoding the whole message

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+decode_word('-- -.--')
+
+def decode(message)
+  msg = ''
+  words = message.split('   ')
+  words.each do |word|
+    msg += "#{decode_word(word)} "
+  end
+  print msg
+end
```
Comment (julie-ify) [inline main.rb]: - [ ] These methods returns `nil` at the end of the decoded morse code. You can fix this issue by removing the `print` statement on line `50` and then add `strip` to the msg returned from the method.  👉  `msg.strip` and the error will go away.

chrome_vipZo802DD

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AdornChoga and @rachidelaid,

Good job so far!

## Highlights!
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (AdornChoga) [reply main.rb]: Thank you @julie-ify. We will do that right away.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @AdornChoga and @rachidelaid,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**